src/components/TextLink/index.tsx

```diff
@@ -0,0 +1,47 @@
+/** @jsx jsx */
+import * as React from "react";
+import { jsx } from "@emotion/core";
+import Text, { FontSizes } from "components/Text";
+import { Theme } from "types/global";
+
+interface Props {
+  href?: string;
```

Comment:
Not sure how accessible this will be if we don't enforce tab-index in lieu of href.

Reply:
So I was thinking about this a little bit, and our major use case for this library is to build the dotcom with it. Next JS uses it's own`<Link />` component to handle routing with an anchor element within in it. That is why I made the href optional.